### PR TITLE
Better addons description

### PR DESCRIPTION
**Pull request by nbessi**
This PR improves the description of the addons and make the names and descriptions more consistent between all Swiss localization modules.

### CODE REVIEW

File: l10n_ch_sepa/__openerp__.py [hunk 1]
```diff
@@ -20,21 +20,23 @@
...
-This module designed for Swiss payment is based on the norm for euro payments
-but implements the special Swiss extensions.
+This module designed for Swiss payment is also generic for euro payments
+as the Swiss standards are slightly different.
 It intends to be reusable to accept new definition of specific standards
-by country.
+by country.  
 
-TODO: Needs to be fully tested with a FI
+It currently supports the "pain.001$2 norm for Credit Transfert Initiation.
```
Comment (yvaucher) [inline l10n_ch_sepa/__openerp__.py:39]: @jgrandguillaume "pain.001$2 is this a typo ?

**Comment by bwrsandman** [comment]
Needs a rebase.

**Comment by nbessi** [comment]
Merge will conflict with flake8 fix

**Comment by nbessi** [comment]
Bad destination branch. 7.0 should be used

**Pull request closed without merging**